python/tvm/relay/build_module.py

```diff
@@ -287,13 +290,17 @@ def _module_export(module, file_name):  # fcompile, addons, kwargs?
 
 
 @register_func("tvm.relay.build")
+def _build_module_no_factory_impl(mod, target, target_host, params, mod_name):
+    target, target_host = Target.check_and_update_host_consist(target, target_host)
+    return build(mod, target, params=params, mod_name=mod_name).module
+
+
```

Comment:
I'm curious that, for what consideration we took the implementation out?

Reply:
To support kwargs